### PR TITLE
feat: supports for local model path

### LINKED ISSUE #82: bug:  Readme issues
### Describe the bug

I detected multiple README flaws. 

1. It should be `w.transcribe(np.ones(16000))` with just one dimension, at least that's what worked for me (and the example afterwards uses `flatten()`, so it is done here.
2. Might be mistaken on this one, but 

> Running transcription on a given Numpy array. This calls full from whisper.cpp. If num_proc is greater than 1, it will use full_parallel instead.

Is simply not the case. In the code in `__init__.py`, `transcribe` always calls `full_parallel`

### To reproduce

Copy paste example README.md code. It fails.

### Expected behavior

It doesn't fail.

### Environment

python: 3.9
plattform: Ubuntu 20.04

### PR DESCRIPTION
**Pull request by sorgfresser**
## What does this PR address?

It adds the ability to specify local filepaths for `from_pretrained` and `from_params` as discussed in #81 .
Additionally, closes #82 

## Before submitting:

- [x] Does the Pull Request follow [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary) naming? Here are [GitHub's
      guide](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) on how to create a pull request.
- [x] Did you read through and follow [development guidelines](../DEVELOPMENT.md)?
- [x] Did your changes require updates to the documentation? Have you updated those accordingly?
- [x] Did you write tests to cover your changes?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,12 +54,13 @@ The binding provides a `Whisper` class:
 from whispercpp import Whisper
 
 w = Whisper.from_pretrained("tiny.en")
+w = Whisper.from_pretrained("/path/to/ggml-model.bin")
```
Comment (aarnphm) [inline README.md:57]: let's keep the upper API docs to have the default alias.

File: README.md [hunk 2]
```diff
@@ -54,12 +54,13 @@ The binding provides a `Whisper` class:
...
 ```
 
 Currently, the inference API is provided via `transcribe`:
 
 ```python
-w.transcribe(np.ones((1, 16000)))
+w.transcribe(np.ones(16000))
```
Comment (aarnphm) [inline README.md:63]: revert

File: README.md [hunk 3]
```diff
@@ -127,12 +128,11 @@ See [DEVELOPMENT.md](./DEVELOPMENT.md)
...
 
-   Running transcription on a given Numpy array. This calls `full` from
-   `whisper.cpp`. If `num_proc` is greater than 1, it will use `full_parallel`
-   instead.
+   Running transcription on a given Numpy array. This calls `full_parallel` from
+   `whisper.cpp`.
 
    ```python
-   w.transcribe(np.ones((1, 16000)))
+   w.transcribe(np.ones(16000))
```
Comment (aarnphm) [inline README.md:135]: same as above.

File: README.md [hunk 4]
```diff
@@ -127,12 +128,11 @@ See [DEVELOPMENT.md](./DEVELOPMENT.md)
 
 2. `Whisper.transcribe(arr: NDArray[np.float32], num_proc: int = 1)`
 
-   Running transcription on a given Numpy array. This calls `full` from
-   `whisper.cpp`. If `num_proc` is greater than 1, it will use `full_parallel`
-   instead.
+   Running transcription on a given Numpy array. This calls `full_parallel` from
+   `whisper.cpp`.
```
Comment (aarnphm) [inline README.md:132]: what is the motivation behind this change? it calls `full` when num_proc is 1

**Review by aarnphm — CHANGES_REQUESTED** [review]
a tiny thing. Can you add a test for this?

There is a folder with all of the model for tests there. Probably just need to have a small unit for this

File: README.md (hunk 1 above)
Comment (sorgfresser) [reply README.md]: Okay! I added a simple comment down in the api docs now and removed this one here.

File: README.md (hunk 2 above)
Comment (sorgfresser) [reply README.md]: Done.

File: README.md (hunk 4 above)
Comment (sorgfresser) [reply README.md]: Yeah, my bad. Reverted.

File: README.md (hunk 3 above)
Comment (sorgfresser) [reply README.md]: Done.

File: tests/export_test.py
Comment (sorgfresser) [inline tests/export_test.py:1]: Added tests now.

**Review by aarnphm — APPROVED** [review]

**Comment by sorgfresser** [comment]
aight, thanks for merging!
